### DISCUSSION TITLE
Support for bot tokens

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#384] [3. Ideas]
I've been using this tool for a while and I'd like to thank you for making such an amazing tool!

So I'm currently in a situation where I need to purge an entire server, but if I did that on my own account, I'd have to set the delay to >7000 ms to reduce my chances of getting banned (and rate limited), but that would take way too long. I wasn't able to find any other tools that were suitable either, and my knowledge isn't good enough to write my own tool. It would be amazing to be able to have it running with a bot token as well.

**Comment by victornpb** [comment]
bots use a different api, but I like the idea. I feel like It will not be as effective or otherwise other bots would offer that kind of feature already... that said I have to do a study to see whats possible.
